pkg/config/compatibility.go

```diff
@@ -327,7 +327,9 @@ func torchvisionGPUPackage(ver string, cuda string) (name string, cpuVersion str
 		}
 	}
 	if latest == nil {
-		return "", "", "", fmt.Errorf("No torchvision GPU package for version %s that's lower or equal to CUDA %s", ver, cuda)
+		// TODO: can we suggest a CUDA version known to be compatible?
+		console.Warnf("Cog doesn't know if CUDA %s is compatible with torchvision %s. This might cause CUDA problems.", cuda, ver)
```

Comment:
style question: could this alternatively be written as `f"Cog doesn't know if CUDA {cuda} is compatible with torchvision {ver}. This might cause CUDA problems."` ?

Reply:
That's Python, unfortunately. Golang doesn't have anything nice like that as far as I know.

Reply:
Oh ha! Didn't even notice. 🙈